src/app/(cms)/[[...slug]]/layout.tsx

```diff
@@ -10,26 +10,20 @@ import HeroBanner from '@/app/components/ui/ukhsa/HeroBanner/HeroBanner'
 import { MegaMenu } from '@/app/components/ui/ukhsa/MegaMenu/MegaMenu'
 import { PhaseBanner } from '@/app/components/ui/ukhsa/PhaseBanner/PhaseBanner'
 import { TopNav } from '@/app/components/ui/ukhsa/TopNav/TopNav'
-import { flags } from '@/app/constants/flags.constants'
 import { getGlobalBanner } from '@/app/hooks/getGlobalBanner'
 import { getServerTranslation } from '@/app/i18n'
-import { getFeatureFlag } from '@/app/utils/flags.utils'
 
 interface LayoutProps {
   children: ReactNode
   params: { slug: string } | null
 }
 
 export default async function Layout({ children, params }: LayoutProps) {
-  const [{ t }, { enabled: landingPageHeroEnabled }, globalBanner] = await Promise.all([
-    getServerTranslation('common'),
-    getFeatureFlag(flags.landingPageHero),
-    getGlobalBanner(),
-  ])
+  const [{ t }, globalBanner] = await Promise.all([getServerTranslation('common'), getGlobalBanner()])
 
-  const { sub_title: subTitle } = await getPageBySlug<PageType.Landing>('landing-page')
+  const onHomePage = !params?.slug
 
-  const onHomePage = landingPageHeroEnabled && !params?.slug
+  const { sub_title: subTitle } = await getPageBySlug<PageType.Landing>('landing-page')
```

Comment:
```suggestion
  const { sub_title: subTitle } = await getLandingPage()
```

We've got a seperate getLandingPage function that can be used here, if you pull latest main though this should be included

Reply:
Actioned